content/en/docs/concepts/storage/volumes.md

```diff
@@ -303,6 +303,12 @@ While tmpfs is very fast, be aware that unlike disks, tmpfs is cleared on
 node reboot and any files you write count against your container's
 memory limit.
 
+{{< note >}}
+If the `SizeMemoryBackedVolumes` feature gate is enabled in 1.20+, users may
```

Comment:
I'd write this:
```suggestion
If the `SizeMemoryBackedVolumes` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled, you can
```

Reply:
Remember to update `/docs/reference/command-line-tools-reference/feature-gates/`